force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -4077,6 +4077,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>bdiContactCustomIdError</shortDescription>
         <value>The Contact Custom Unique ID setting field {0} must have matching fields in the NPSP Data Import object whose API Names are {1} and {2}.</value>
     </labels>
+    <labels>
```

Comment:
Do we need to tag Erin here for a review?

Reply:
@callen-sfdo I sent you some questions in your WI Chatter feed. I need a little more info on this work and I'll have to pass all UI text through the UI text review team next week.